src/contexts/TodoContext.js

```diff
@@ -0,0 +1,66 @@
+import React, { createContext, useState } from 'react'
+
+const TodoContext = createContext()
+
+const TodoProvider = ({ children }) => {
+  const [todos, setTodos] = useState([])
+  const [todo, setTodo] = useState('')
+  const [statusTodo, setStatusTodo] = useState('all')
```

Comment:
Para hacer más claro que refiere al filtro, se podría indicar:

```suggestion
  const [statusTodoFilter, setStatusTodoFilter] = useState('all')
```